src/components/encoding-pane/property-editor-schema.ts

```diff
@@ -58,6 +58,9 @@ export interface PropertyEditorSchema {
   schema: ObjectSchema;
 }
 
+// Currently supported customizble encoding channels that display caret in customizer UI
+export const CUSTOMIZABLE_ENCODING_CHANNELS = [Channel.X, Channel.Y, Channel.COLOR, Channel.SIZE, Channel.SHAPE];
```

Comment:
How about row and column?

Reply:
Those are not supported as of now. Eventually it'd be nice to customize the row/column encoding channels. It'd be nice to support customizing headers and whatnot. 

So better just leave what is working right now, and then when add support for row/column, update this list as well